src/modules/Users.pm

```diff
@@ -3461,7 +3172,6 @@ BEGIN { $TYPEINFO{SubstituteUserValues} = ["function", "void"] }
 sub SubstituteUserValues {
 
     my $self	= shift;
-    my $substituted = UsersLDAP->SubstituteValues ("user", \%user_in_work);
```

Comment:
Here the declaration is removed, but not the use of $substituted in the following lines.

Reply:
it looks like maye whole method can be dropped?